include/mitsuba/core/bitmap.h

```diff
@@ -182,33 +182,34 @@ class MTS_EXPORT_CORE Bitmap : public Object {
     // ======================================================================
 
     /**
-     * \brief Create a bitmap of the specified type and allocate
-     * the necessary amount of memory
+     * \brief Create a bitmap of the specified type and allocate the necessary
+     * amount of memory
      *
-     * \param pixel_format
-     *    Specifies the pixel format (e.g. RGBA or Luminance-only)
+     * \param pixel_format Specifies the pixel format (e.g. RGBA or
+     *    Luminance-only)
      *
-     * \param component_format
-     *    Specifies how the per-pixel components are encoded
-     *    (e.g. unsigned 8 bit integers or 32-bit floating point values).
-     *    The component format struct_type_v<Float> will be translated to the
-     *    corresponding compile-time precision type (Float32 or Float64).
+     * \param component_format Specifies how the per-pixel components are
+     *    encoded (e.g. unsigned 8 bit integers or 32-bit floating point
+     *    values). The component format struct_type_v<Float> will be translated
+     *    to the corresponding compile-time precision type (Float32 or Float64).
      *
-     * \param size
-     *    Specifies the horizontal and vertical bitmap size in pixels
+     * \param size Specifies the horizontal and vertical bitmap size in pixels
      *
-     * \param channel_count
-     *    Channel count of the image. This parameter is only required when
-     *    \c pixel_format = \ref PixelFormat::MultiChannel
+     * \param channel_count Channel count of the image. This parameter is only
+     *    required when \c pixel_format = \ref PixelFormat::MultiChannel
      *
-     * \param data
-     *    External pointer to the image data. If set to \c nullptr, the
+     * \param channel_names Channel names of the image. This parameter is
+     *    optional, and only used when \c pixel_format = \ref
+     *    PixelFormat::MultiChannel
+     *
+     * \param data External pointer to the image data. If set to \c nullptr, the
```

Comment:
For consistency, I would prefer keeping the new line after the parameter name (new line for the description)